config/locales/gobierto_data/views/ca.yml

```diff
@@ -41,6 +41,8 @@ ca:
       runQuery: Executar consulta
```

Comment:
I gonna send you a commit here for the translations. I'll auto solve this conversation

Reply:
Done